setup.py

```diff
@@ -96,7 +96,7 @@
         'tweakwcs>=0.5.0',
         'stregion',
         'requests',
-        # HLA-pipeline specific:
+        # EPP-pipeline specific:
```

Comment:
What is EPP?

Reply:
@pllim https://github.com/spacetelescope/drizzlepac/pull/312#discussion_r273654042

Reply:
Just an FYI as an alternative to EPP (maybe).  The new products being produced are being called the Hubble Advanced Products (HAP) in the system databases.

Reply:
Whatever you or MO decide. Just let me know.